stories/topics/tech-note-aws-remove-extension-node.adoc

```diff
@@ -0,0 +1,5 @@
+[id="tech-note-aws-remove-extension-node"]
+
+= Remove extension node note
+
+When removing extension node using ops playbook, make sure you have provided correct autoscaling group name and launch template name. Incorrect autoscaling group name and launch template name will result in some orphan resources.
```

Comment:
using the (Ansible-on-Clouds?)ops playbook,

launch template name results in some...